src/EditorFeatures/CSharpTest/UsePatternCombinators/CSharpUsePatternCombinatorsDiagnosticAnalyzerTests.cs

```diff
@@ -125,6 +124,13 @@ public async Task TestOnExpression(string expression, string expected)
             await TestAllOnExpressionAsync(expression, expected);
         }
 
+        [InlineData("nullable == 1 || 2 == nullable", "nullable is 1 or 2")]
+        [Theory(Skip = "https://github.com/dotnet/roslyn/issues/56938"), Trait(Traits.Feature, Traits.Features.CodeActionsUsePatternCombinators)]
```

Comment:
this case regressed.  we now produce `nullable is (int?)1 or (int?)2`.  We'll need special rules to simplify casts in pattersn.  This is tracked in https://github.com/dotnet/roslyn/issues/56938